e2e/tests/truffle/test/EthFunctions.js

```diff
@@ -144,5 +145,38 @@ contract('MyToken', async (accounts) => {
     assert.equal(ethOwner.toLowerCase(), web3js.utils.padLeft(owner, 64).toLowerCase(), "result using tokenContract and eth.call");
   });
 
-});
+  it('MyToken eth_estimateGas', async () => {
```

Comment:
One test case is enough I think, you can remove this one and keep the one below.